d4tools/Dockerfile

```diff
@@ -4,13 +4,13 @@
 
 FROM rust:latest AS builder
 
-RUN cargo install d4tools
+RUN cargo install --git https://github.com/38/d4-format.git d4tools --branch master
 
 #########
 # FINAL #
 #########
 
-FROM debian:bullseye-slim
+FROM debian:bookworm-slim
```

Comment:
This is fixing the issue for this image:

<img width="954" alt="image" src="https://github.com/user-attachments/assets/d8d77adc-1396-44f6-a5e1-b74fc178d2cf">